integrations/docker/images/chip-build/version

```diff
@@ -1 +1 @@
-0.5.33 Version bump reason: [Ameba] Lwip update
+0.5.34 Version bump reason: Zap docker image creation
```

Comment:
this is a brand new image with a new version, version does not need to be updated